src/main/java/com/jfrog/ide/idea/ui/JFrogToolWindow.java

```diff
@@ -19,15 +20,22 @@ public class JFrogToolWindow {
 
     void initToolWindow(@NotNull ToolWindow toolWindow, @NotNull Project project, boolean buildsConfigured) {
         ContentManager contentManager = toolWindow.getContentManager();
-        JFrogLocalToolWindow jfrogLocalContent = new JFrogLocalToolWindow(project);
+        JFrogLocalToolWindow jfrogLocalContent = null;
+        try {
+            jfrogLocalContent = new JFrogLocalToolWindow(project);
+        } catch (Exception e) {
+            Logger.getInstance().error("Local view could not be initialized.", e);
```

Comment:
This is troublesome. In that case, we may like to get an exception. This log message is nearly invisible.

Reply:
I added an empty state for this case.